src/middleware/jira-symmetric-jwt-middleware.ts

```diff
@@ -130,8 +143,11 @@ const checkPathValidity = (url: string) => {
 	});
 };
 
-const checkGenericContainerActionUrl = (url: string) => {
-	return genericContainerActionUrls.some(moduleUrl => {
+const checkGenericContainerActionUrl = async (url: string) => {
```

Comment:
/nit much of a muchness but Im feeling commenty

`  
return genericContainerActionUrls?.some(moduleUrl => matchRouteWithPattern(moduleUrl, url)) ?? false;
`